package.json

```diff
@@ -56,7 +56,7 @@
     "mocha": "^5.2.0",
     "nock": "^9.4.1",
     "node-mocks-http": "^1.7.0",
-    "nyc": "^12.0.2",
+    "nyc": "^11.7.0",
```

Comment:
Out of curiosity, why was `nyc` downgraded in this PR?

Reply:
Yeah, that's the problem. Upgrading nyc breaks istanbul, because of a semver.major bump breakage in randomatic. (don't say I haven't foresaid this as a problem)

Reply:
Wait, that doesn't make sense.. `nyc` is just a wrapper around istanbul, and they release in lockstep pretty much. What do you mean by it breaks istanbul?

Reply:
scroll up the backlog, it explains it better than I can do (I don't really understand it, though I'd like to :-) )

Reply:
Ahhhh I see! :) I'm sorry, I totally missed those parts.